keyring_mock.go

```diff
@@ -50,6 +62,38 @@ func (m *mockProvider) Delete(service, user string) error {
 	return ErrNotFound
 }
 
+// Unlock unlocks item from the keyring given a service name and a user
+func (m *mockProvider) Unlock(service, user string) error {
```

Comment:
I'm not sure about the value of adding this to the mockProvider. It won't be testing the change in the unix implementation. What is the idea behind it?

Reply:
yea, I agree, I think it's unnecessary. I thought this mock provider is for test cases and not public to everyone. So, I wanted to call unlock before accessing something. But looks useless to me. I'll remove it. Thanks!